libs/barista-components/button-group/src/button-group.ts

```diff
@@ -194,24 +203,44 @@ export class DtButtonGroupItem<T> extends _DtButtonGroupItem
     HasTabIndex,
     AfterContentInit,
     OnDestroy {
-  private _selected = false;
+  private _checked = false;
   private _value: T;
   private _disabled = false;
 
-  /** Emits a stream when this item is selected or deselected. */
+  /**
+   * @deprecated
+   * @breaking-change will be removed with version 9.0.0 please use the `checkedChange` instead.
+   */
   @Output() readonly selectionChange = new EventEmitter<
     DtButtonGroupItemSelectionChange<T>
   >();
 
-  /** Whether the button-group item is selected. */
+  /** Emits a stream when this item is checked or unchecked. */
+  @Output() readonly checkedChange = new EventEmitter<
+    DtButtonGroupItemCheckedChange<T>
+  >();
+
+  /**
+   * @deprecated
+   * @breaking-change will be removed with version 9.0.0 please use the `checked` property instead.
+   */
   @Input()
   get selected(): boolean {
-    return this._selected && !this.disabled;
+    return this._checked;
```

Comment:
Please proxy the existing getter. Otherwise you wont have the full fledged check with the disabled

```suggestion
    return this.checked;
```